### DISCUSSION TITLE
how can I join the project

### DISCUSSION THREAD

**Discussion by Med-H** [MichaelCade/90DaysOfDevOps#23] [General]
Hi Mike, can I join the project?

thank you in advance

**Comment by MichaelCade** [comment]
Hey, thanks for starting a discussion topic. 

There are several ways to get involved. Either by adding input into the days I have completed or correcting something I have noted that isn't true. I would be open to also adding great free resources that I have not covered. 

Or you can fork the project and work through yourself using the resources noted on each topic. 

The premise of the project is to document and learn in public my journey to learning more about DevOps processes, tools and concepts. 

A lot of people have forked the repo and using in their own ways. 

I am also thinking that I should make a blank template of the repo which people can use and document their own learning journey. 

Hope that helps?
